.drone.yml

```diff
@@ -0,0 +1,12 @@
+image: bradrydzewski/base
+cache:
+  - /var/run
```

Comment:
This is a HACK to workaround the lack of Docker volumes support, to allow the container to use the host Docker.